contentcuration/contentcuration/api.py

```diff
@@ -20,18 +20,18 @@ def write_file_to_storage(fobj, check_valid = False):
     checksum = hashlib.md5()
     for chunk in iter(lambda: fobj.read(4096), b""):
         checksum.update(chunk)
-    filename, ext = os.path.splitext(fobj._name)
+    filename, ext = os.path.splitext(fobj._name) if fobj._name is not None else ("", "")
```

Comment:
This will work as long as `_name` is actually a defined attribute. Should be ok for now, but an even safer way would be `if getattr(fobj, "_name", None) is not None`.